ios/RNIapIosSk2.swift

```diff
@@ -848,6 +867,30 @@ class RNIapIosSk2iOS15: Sk2Delegate {
         #endif
     }
 
+    @objc public func showManageSubscriptions(
+        _ resolve: @escaping RCTPromiseResolveBlock = { _ in },
+        reject: @escaping RCTPromiseRejectBlock = { _, _, _ in }
+    ) {
+        #if !os(tvOS)
+        guard let scene = UIApplication.shared.connectedScenes.first as? UIWindowScene,
```

Comment:
Is this a better way of getting the scene? I was using `await UIApplication.shared.keyWindow?.windowScene` on `beginRefundRequest` line :915

Reply:
Not sure, maybe this is faster as it isn't async? But I'm not a Swift/Objective-C expert